server_admin/topics/vault.adoc

```diff
@@ -1,46 +1,45 @@
+
 [[_vault-administration]]
 
 == Using Vault to Obtain Secrets
 
-Several fields in the administration support obtaining the value of a secret from an external vault.
+To obtain a secret from a vault rather than entering it directly, enter the following specially crafted string into the appropriate field:
 
-To obtain a secret from a vault instead of entering it directly, enter
-the following specially crafted string into the appropriate field:
-`**${vault.**_key_**}**` where you replace the `_key_`
-with the name of the secret as recognized by the vault.
+[source]
+----
+**${vault.**_key_**}**
+----
+where the `_key_` is the name of the secret recognized by the vault.
 
-In order to prevent secrets from leaking across realms, implementations may combine the realm name with the `_key_`
-obtained from the vault expression. This means that the `_key_` won't directly map to an entry in the vault, but rather
-be used to create the final entry name according to the algorithm used to combine it with the realm name.
+{project_name} combines the realm name with the `_key_` obtained from the vault expression to prevent secrets from leaking across realms. This method means that the `_key_` does not directly map to an entry in the vault but creates the final entry name according to the algorithm used to combine the `_key_` with the realm name.
```

Comment:
Suggest reversing to lead with "To prevent secrets from leaking across realms, ..."